app/bundles/EmailBundle/Views/Email/recipients.html.php

```diff
@@ -0,0 +1,21 @@
+<form novalidate="" autocomplete="false" data-toggle="ajax" role="form" name="email" method="post" action="<?= $action; ?>"> 
+
+    <div class="row">
+        <div class="form-group col-xs-10 ">
+            <label class="control-label" >Recipients</label>
```

Comment:
These should be translated - although I'm not sure how I feel about the inconsistency of using raw form output here where we use a form type class everywhere else in Mautic. https://developer.mautic.org/#forms